engine/server/sv_init.c

```diff
@@ -276,11 +276,20 @@ model_t *SV_ModelHandle( int modelindex )
 	return sv.models[modelindex];
 }
 
+static resourcetype_t SV_DetermineResourceType( const char *filename )
+{
+	if( !Q_strncmp( filename, "sound/", 6 ) && Sound_SupportedFileFormat( COM_FileExtension( filename )))
```

Comment:
You can use `DEFAULT_SOUNDPATH` macro here:
```
if( !Q_strncmp( filename, DEFAULT_SOUNPATH, sizeof( DEFAULT_SOUNPATH ) - 1 ) && Sound_SupportedFileFormat( COM_FileExtension( filename )))
```

macro:
https://github.com/FWGS/xash3d-fwgs/blob/master/engine/common/com_strings.h#L50=